Common/app/ota/ota_update_task.c

```diff
@@ -144,12 +144,15 @@
  * This topic filter is used to match the responses from OTA service for OTA agent job requests. THe
  * topic filter is a reserved topic which is not subscribed with MQTT broker.
  */
-#define OTA_JOB_ACCEPTED_RESPONSE_TOPIC_FILTER    OTA_TOPIC_PREFIX "/+/jobs/$next/get/accepted"
+#define OTA_JOB_ACCEPTED_RESPONSE_TOPIC_FILTER    OTA_TOPIC_PREFIX "/+/jobs/$next/get/#"
 
 /**
  * @brief Wildcard topic filter for matching OTA job update messages (used to detect cancellation).
  */
-#define OTA_JOB_UPDATE_TOPIC_FILTER               OTA_TOPIC_PREFIX "/+/jobs/+/update/#"
+#define OTA_JOB_UPDATE_TOPIC_FILTER               OTA_TOPIC_PREFIX "/+/jobs/+/update/rejected"
```

Comment:
Is this topic filter being used anywhere ? Also the topic filter name does not match the definition.